docs/src/interfacer.md

```diff
@@ -138,18 +139,26 @@ This function is expected to be extended for the
 following properties, and may also be extended for any additional
 properties needed by a component model.
 
-| Coupler name      | Description | Units |
-|-------------------|-------------|-------|
-| `emissivity`   | surface emissivity | |
-| `surface_direct_albedo`   | bulk direct surface albedo over the whole surface space | |
-| `surface_diffuse_albedo`   | bulk diffuse surface albedo over the whole surface space | |
-| `surface_temperature` | temperature over the combined surface space | K |
-| `turbulent_fluxes` | turbulent fluxes (note: only required when using `PartitionedStateFluxes` option - see our `FluxCalculator` module docs for more information) | W m⁻² |
+| Coupler name             | Description                                              | Units |
+|--------------------------+----------------------------------------------------------+-------|
+| `emissivity`             | surface emissivity                                       |       |
+| `surface_direct_albedo`  | bulk direct surface albedo over the whole surface space  |       |
+| `surface_diffuse_albedo` | bulk diffuse surface albedo over the whole surface space |       |
+| `surface_temperature`    | temperature over the combined surface space              | K     |
+| `turbulent_fluxes`       | turbulent fluxes                                         | W m⁻² |
 
 - `calculate_surface_air_density(atmos_sim::Interfacer.AtmosModelSimulation, T_sfc::ClimaCore.Fields.Field)`:
 A function to return the air density of the atmosphere simulation
 extrapolated to the surface, with units of [kg m⁻³].
 
+ClimaAtmos should also add the following coupler fields for Monin-Obukhov similarity theory:
+| Coupler name    | Description       | Units  |
+|-----------------+-------------------+--------|
+| `ustar`         | friction velocity | m s⁻¹  |
+| `L_MO`          | Obukhov length    | m      |
+| `buoyancy_flux` | flux of buoyancy  | m⁻²s⁻³ |
```

Comment:
Is this specific to ClimaAtmos, or will any atmosphere model need to provide these? If it's any atmosphere model, we should generalize to `AtmosModelSimulation`

Reply:
I think they are required if you use the default FluxCalculator function, but you can probably provide methods to not use them.